### PR TITLE
[AJ-1491][AJ-1492] Update billing project/workspace cost notices

### PR DESCRIPTION
**Pull request by nawatts**
https://broadworkbench.atlassian.net/browse/AJ-1491
https://broadworkbench.atlassian.net/browse/AJ-1492

### CODE REVIEW

File: src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/CreateNamedProjectStep.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ const AzureCostWarnings = () => {
     div({ style: rowEntry }, [
       icon('warning-standard', { size: 16, style: { marginRight: '0.5rem', color: colors.warning() } }),
       div([
-        'Creating a Terra on Azure Billing Project has a starting cost of $10.58/day and increases with usage. ',
+        'A Terra on Azure Billing Project has a starting cost of $10-15/day depending on region. ',
```
Comment (bethsheets) [inline src/pages/billing/NewBillingProjectWizard/AzureBillingProjectWizard/CreateNamedProjectStep.ts]: oops, actually, can we say: "A Terra on Azure Billing Project has a starting cost of $10-15/day, depending on the region you select, and increases with usage."

**Review by bethsheets — CHANGES_REQUESTED** [review]
One small change. Thank you!

**Review by bethsheets — APPROVED** [review]
looks good, thank you!

**Review by cahrens — APPROVED** [review]

**Review by jladieu — APPROVED** [review]
Approved as long as @bethsheets likes the prose!

**Review by calypsomatic — APPROVED** [review]

**Pull request merged**